### PR TITLE
Fixed Code Quality in Recording page

### PR DESCRIPTION
**Pull request by Axelh04**
**Description:**

- Fixed component structure in Recording Page
- Removed unnecessary comments
- Removed Redundant Code
- Fixed use of setTimeout functions
- Changed stylesheet names
- Removed unused props (unless noted)
- Fixed audio not showing full time bug
- Removed fetching image assets from the internet

**Next Steps:**

- Profile Page Code Quality Passthrough
- Fix single note detection for Similar Sounds API request
- Extra UX
- ReadME

**Test Plan:**

Audio bug fix, now shows length of recording:
image

### CODE REVIEW

File: frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx [hunk 1]
```diff
@@ -106,6 +99,10 @@ function PlaybackContainer({
     return pitches;
   }
 
+  // Takes in audio buffer data and performs an operation that compares two versions of the audio data
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx:102]: Just a tip, you can use /* */ for multi-line comments

File: frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx [hunk 2]
```diff
@@ -106,6 +99,10 @@ function PlaybackContainer({
...
+  // Uses lag between wavelength data to determine when a vocal pitch is detected, if so this returns that frequency in hz
+  // Performs function at specific slice determined by the analyzePitchOverTime algorithim
+  // Inspired by Yin's mathematical algorithim for pitch detection
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx:105]: Just a tip, you can use /* */ for multi-line comments

File: frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx [hunk 3]
```diff
@@ -69,25 +76,36 @@ function RecordingSelector({ setSelectedBlob, selectedBlob, setIsLoadingRecordin
...
     }
   };
 
+  useEffect(() => {
+    const newBlobs = recordings.map((item) => {
+      return URL.createObjectURL(item);
+    });
+    setRecordingUrls(newBlobs);
+
+    return () => {
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx:95]: What does this do?

File: frontend/src/RecordingPageComp/SimilarSounds/SimilarSounds.jsx [hunk 4]
```diff
@@ -9,20 +9,23 @@ function SimilarSounds({ playbackDuration }) {
...
+    //Freesound API request
     try {
+      //Loading State
+      setIsLoadingSounds(true);
       const API_KEY = import.meta.env.VITE_APP_API_KEY;
       const response = await fetch(
         `https://freesound.org/apiv2/search/text/?token=${API_KEY}&filter=duration:${duration}&fields=name,previews`
       );
+      if (!response.ok) {
+        throw new Error(`HTTP error! Status: ${response.status}`);
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/SimilarSounds/SimilarSounds.jsx:21]: You may want to set loading state to false here too or in the catch block. Execution of the try block will stop when an error is thrown and continue in the catch block. So, the loading state will stay set to True when an error occurs. As we discussed offline, I would make sure to test all code paths to ensure behavior is expected in both success and failure cases.

File: frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx [hunk 5]
```diff
@@ -64,18 +62,13 @@ function PlaybackContainer({
...
 
-              //Single Pitch detector
-              const singlePitchDetector = PitchDetector.forFloat32Array(
-                input.length
-              );
-              const [pitch] = singlePitchDetector.findPitch(
-                input,
-                audioContextRef.current.sampleRate
-              );
+              //TODO Function to calculate average MIDInotes in filtered pitches to get the average pitch for API request
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx:68]: nice TODO!

**Comment by kfullen** [comment]
Really like the use of comments now!

File: frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx [hunk 6]
```diff
@@ -69,25 +76,36 @@ function RecordingSelector({ setSelectedBlob, selectedBlob, setIsLoadingRecordin
...
+      newBlobs.forEach((blobUrl) => URL.revokeObjectURL(blobUrl));
+    };
+  }, [recordings]);
+
```
Comment (kfullen) [inline frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx:99]: What does this do?

File: frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx (hunk 2 above)
Comment (Axelh04) [reply frontend/src/RecordingPageComp/PlaybackContainer/PlaybackContainerComp.jsx:105]: Noted, will fix this from now on.

File: frontend/src/RecordingPageComp/SimilarSounds/SimilarSounds.jsx (hunk 4 above)
Comment (Axelh04) [reply frontend/src/RecordingPageComp/SimilarSounds/SimilarSounds.jsx:21]: Will work on this when performing testing out paths that lead to an error.

File: frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx (hunk 6 above)
Comment (Axelh04) [reply frontend/src/RecordingPageComp/RecordingSelection/RecordingSelectionComp.jsx:99]: As discussed in 1:1, removes URL from each blob to improve performance.

**Pull request merged**